client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -1,123 +1,59 @@
-import { connect } from 'react-redux';
 import wpToJpImageRtl from 'calypso/assets/images/jetpack/wp-to-jp-rtl.svg';
 import wpToJpImage from 'calypso/assets/images/jetpack/wp-to-jp.svg';
 import AppsBadge from 'calypso/blocks/get-apps/apps-badge';
 import CardHeading from 'calypso/components/card-heading';
 import userAgent from 'calypso/lib/user-agent';
-import { recordTracksEvent, withAnalytics } from 'calypso/state/analytics/actions';
-import { sendEmailLogin } from 'calypso/state/auth/actions';
-import { getCurrentUserEmail } from 'calypso/state/current-user/selectors';
 
 import './style.scss';
 
-export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
+export const GoMobile = () => {
 	const isRtl = useRtl();
 	const translate = useTranslate();
 	const localizeUrl = useLocalizeUrl();
 	const { isDesktop, isiPad, isiPod, isiPhone, isAndroid } = userAgent;
 	const isIos = isiPad || isiPod || isiPhone;
-	const isDesktopApp = config.isEnabled( 'desktop' );
-	const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
 
-	const emailLogin = () => {
-		sendMobileLoginEmail( email );
-	};
-
-	if ( displayJetpackAppBranding ) {
-		const showIosBadge = ! isAndroid && ! isDesktop;
-		const showAndroidBadge = ! isIos && ! isDesktop;
-		const showBadge = showIosBadge || showAndroidBadge;
-
-		return (
-			<Card className="go-mobile go-mobile--jetpack customer-home__card">
-				<div className="go-mobile__row">
-					<img
-						className="go-mobile__icon"
-						src={ isRtl ? wpToJpImageRtl : wpToJpImage }
-						width="49"
-						height="29"
-						alt="WordPress and Jetpack app"
-					/>
-					<div className="go-mobile__title">
-						<CardHeading tagName="h2">{ translate( 'Get our mobile app' ) }</CardHeading>
-						<h3 className="go-mobile__subheader">
-							{ translate( 'Everything you need to build and grow your site from any device.' ) }
-						</h3>
-					</div>
-					{ showBadge ? (
-						<div className="go-mobile__app-badges">
-							<AppsBadge
-								storeName={ showIosBadge ? 'ios' : 'android' }
-								utm_source="calypso-customer-home"
-							></AppsBadge>
-						</div>
-					) : (
-						<Button
-							className="go-mobile__email-link-button is-link"
-							href={ localizeUrl(
-								'https://apps.wordpress.com/get/?campaign=calypso-customer-home'
-							) }
-						>
-							{ translate( 'Get the Jetpack app' ) }
-						</Button>
-					) }
-				</div>
-			</Card>
-		);
-	}
-
-	const showIosBadge = ! isAndroid;
-	const showAndroidBadge = ! isIos;
-	const showOnlyOneBadge = showIosBadge !== showAndroidBadge;
+	const showIosBadge = ! isAndroid && ! isDesktop;
+	const showAndroidBadge = ! isIos && ! isDesktop;
+	const showBadge = showIosBadge || showAndroidBadge;
 
 	return (
 		<Card className="go-mobile customer-home__card">
-			<div className={ classnames( 'go-mobile__row', { 'has-2-cols': showOnlyOneBadge } ) }>
+			<div className="go-mobile__row">
```

Comment:
Seems like this `className` is no longer necessary since it has no styles applied to it. Should we remove it? Potentially also the entire `div`?

Reply:
Ah, great point, thank you! Removed the `div` in https://github.com/Automattic/wp-calypso/pull/74290/commits/1992c7612e1c3bcf08db288516b1711922eed2b0.